src/app/services/dimBucketService.factory.js

```diff
@@ -69,7 +69,7 @@ function BucketService(dimDefinitions, dimCategory) {
     BUCKET_MODS: "Ornaments",
     BUCKET_EMOTES: "Emote",
     BUCKET_MAIL: "Messages",
-    BUCKET_BUILD: "Class",
+    BUCKET_BUILD: "Subclass",
```

Comment:
Might want to leave this and make D2 use Class, I don't know where else we use Class.